src/pilot/deploy-overcloud.py

```diff
@@ -579,6 +586,14 @@ def main():
         if args.enable_unity_manila:
             env_opts += " -e ~/pilot/templates/unity-manila-config.yaml"
 
+        if args.enable_powermax:
+           if self.settings.powermax_storage_protocol == "iSCSI":
```

Comment:
Deploy-overcloud.py will not resolve self.settings.powermax_storage_protocol, you probably want to pass which protocol to use as another param instead

Reply:
Fixed it